### PR TITLE
Trip Options Mode Selector Bug Fixes

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
This PR comprises different bug fixes in the new TripOptions component.

### CODE REVIEW

**Comment by miles-grant-ibigroup** [comment]
Thanks @grant-humphries for figuring out how to reproduce this! This is now fixed.

CI tests are failing pending the `core-utils` being separated into a second PR, which will happen once this PR is approved.

File: packages/core-utils/src/query-params.js
Comment (grant-humphries) [inline packages/core-utils/src/query-params.js:270]: This language is kind of niche so I feel like the text for these labels should come from the config file.  Also our marketing team likes to change this kind of thing often so would be helpful in that sense too.  Don't see this as blocking to merge this code though

**Review by grant-humphries — APPROVED** [review]
I'd recommend changing the default mode in the TripOption story to some kind of transit only variant because that's a common setting for users and it has caused (no resolved) issues that can't be reproduced when the staring value in walk only

**Comment by miles-grant-ibigroup** [comment]
Thanks @grant-humphries! Good call with the default params, have updated this. As to how the strings will be added, I will defer to the rest of my team. Thanks for all your help and guidance!

Assigning to @binh-dam-ibigroup

File: packages/core-utils/src/query-params.js
Comment (binh-dam-ibigroup) [reply packages/core-utils/src/query-params.js:270]: Customization of these strings is already covered by https://github.com/opentripplanner/otp-ui/pull/273.

File: packages/trip-form/src/TripOptions/OptionButton.tsx
Comment (binh-dam-ibigroup) [inline packages/trip-form/src/TripOptions/OptionButton.tsx:8]: Deserves prop sorting.

File: packages/trip-form/src/test-utils/modes.js
Comment (binh-dam-ibigroup) [inline packages/trip-form/src/test-utils/modes.js:131]: These deletions of the wheelchair setting will affect demos involving accessibility. Is that intended?

**Review by binh-dam-ibigroup — COMMENTED** [review]
@grant-humphries, could you advise regarding string customization on your implementation using functionality introduced by #273, and @miles-grant-ibigroup, could you advise regarding removing some of the accessibility demo params.

File: packages/trip-form/src/test-utils/modes.js
Comment (miles-grant-ibigroup) [reply packages/trip-form/src/test-utils/modes.js:131]: You are right we can probably bring wheelchairs back for some modes!

Comment (miles-grant-ibigroup) [reply packages/trip-form/src/test-utils/modes.js:131]: It's in the car modes which still allows the setting to be tested.

File: packages/trip-form/src/TripOptions/OptionButton.tsx
Comment (miles-grant-ibigroup) [reply packages/trip-form/src/TripOptions/OptionButton.tsx]: Sorted in c58e1ba1b18a029d706b79ede998dd3b486e1d21

**Comment by miles-grant-ibigroup** [comment]
Strings outside of `trip-form` have been reverted as to not cause compatibility issues with others using the project. @grant-humphries as Binh said you should be able to use internationalization to replace these strings.

**Comment by grant-humphries** [comment]
@binh-dam-ibigroup @miles-grant-ibigroup sounds good with respect the label strings, our team will plan to get things wired up via internationalization this week and let you know how it goes

File: packages/trip-form/src/TripOptions/OptionButton.tsx
Comment (binh-dam-ibigroup) [inline packages/trip-form/src/TripOptions/OptionButton.tsx:8]: Props need to be sorted.

**Review by binh-dam-ibigroup — APPROVED** [review]
One place to sort props and it's good to go.

File: packages/trip-form/src/TripOptions/OptionButton.tsx
Comment (miles-grant-ibigroup) [reply packages/trip-form/src/TripOptions/OptionButton.tsx]: Sorted in 
62634db

**Review by philip-cline — APPROVED** [review]

**Pull request merged**
